Integrations/integration-EWSv2.yml

```diff
@@ -143,6 +143,7 @@ script:
     LAST_RUN_TIME = "lastRunTime"
     LAST_RUN_IDS = "ids"
     LAST_RUN_FOLDER = "folderName"
+    ERROR_COUNTER = "errors"
```

Comment:
@BenJoParadise please change the value to `errorCounter`/`errorAmount` - something that would match the naming